Cargo.toml

```diff
@@ -25,11 +25,12 @@ all-features = true
 [dependencies]
 libloading = { version = "0.8", optional = true }
 once_cell = { version = "1.19", optional = true }
-windows = { version = ">=0.56, <=0.58", features = ["implement", "Win32_Foundation", "Win32_Graphics_Direct3D12", "Win32_Storage_FileSystem"] }
-windows-core = { version = ">=0.56, <=0.58" }
+windows = { version = ">=0.57, <=0.59", features = ["Win32_Foundation", "Win32_Graphics_Direct3D12", "Win32_Storage_FileSystem"], default-features = false }
+windows-core = ">=0.57, <=0.59"
+windows-targets = "0.53"
```

Comment:
@hasenbanck I'm sorry, I completely forgot to revisit this. This new `windows-targets` version is only used since `windows 0.59` meaning users will get a duplicate on older `windows(-core)` versions, even though `direct-storage` can also compile against `windows-targets 0.52` to get rid of the duplicate. Solved in #27.